frontend_tests/puppeteer_tests/drafts.ts

```diff
@@ -103,7 +132,7 @@ async function test_previously_created_drafts_rendered(page: Page): Promise<void
             page,
             ".draft-row .message_header_private_message .stream_label",
         ),
-        "You and Cordelia, Lear's daughter, King Hamlet",
+        "You and King Hamlet, Cordelia, Lear's daughter",
```

Comment:
I'm not totally sure why this was switched around, but my guess was that by actually sending a message to the stream instead of having this recipient only in the draft, the order got sorted/swapped?